packages/rolldown/src/utils/validator.ts

```diff
@@ -844,7 +843,7 @@ export function validateOption<T>(key: 'input' | 'output', options: T): void {
       })
       .filter(Boolean);
     if (errors.length) {
-      throw new Error(`Failed validate ${key} options.\n` + errors.join('\n'));
+      console.warn(`Warning validate ${key} options.\n` + errors.join('\n'));
```

Comment:
If `ROLLDOWN_OPTIONS_VALIDATION` is `strict`, should we throw errors?

Reply:
The strict  mode is droped, it is unnecessary after loose mode + warning